src/libs/NetworkConnection.ts

```diff
@@ -72,42 +73,51 @@ Onyx.connect({
  * internet connectivity or not. This is more reliable than the Pusher
  * `disconnected` event which takes about 10-15 seconds to emit.
  */
-function subscribeToNetInfo(): void {
-    // Note: We are disabling the configuration for NetInfo when using the local web API since requests can get stuck in a 'Pending' state and are not reliable indicators for "offline".
+function subscribeToNetInfo() {
+    let backendReachabilityCheckInterval: NodeJS.Timeout;
+    // Note: We are disabling the reachability check when using the local web API since requests can get stuck in a 'Pending' state and are not reliable indicators for "offline".
     // If you need to test the "recheck" feature then switch to the production API proxy server.
     if (!CONFIG.IS_USING_LOCAL_WEB) {
-        // Calling NetInfo.configure (re)checks current state. We use it to force a recheck whenever we (re)subscribe
-        NetInfo.configure({
-            // By default, NetInfo uses `/` for `reachabilityUrl`
-            // When App is served locally (or from Electron) this address is always reachable - even offline
+        // Set interval to (re)check current state every 60 seconds
```

Comment:
I don't think we should extract something as a constant and then use its literal value in a comment. This is a recipe for a desync...